src/ol/view.js

```diff
@@ -815,7 +821,7 @@ ol.View.prototype.getZoomForResolution = function(resolution) {
   var max, zoomFactor;
   if (this.resolutions_) {
     var nearest = ol.array.linearFindNearest(this.resolutions_, resolution, 1);
-    offset += nearest;
+    offset = nearest;
```

Comment:
The `offset` variable is no longer used.

Reply:
`offset` is used when it returns.

Reply:
Right :smile: - had missed that in the diff view.